### PR TITLE
Add custom scripts error metric filters to cloudwatch dashboard

### PR DESCRIPTION
**Pull request by chenwany**
### Description of changes
* Add Custom script error metric filters for "OnNodeStartDownloadError", "OnNodeStartExecutionError", "OnNodeConfiguredDownloadError", "OnNodeConfiguredExecutionError"
* 
image

### Tests
* Unit test
* Manually test

### References
* Link to related PRs in other packages (i.e. cookbook, node).
* Link to documentation useful to understand the changes.

### Checklist
- [ ] Make sure you are pointing to **the right branch** and add a label in the PR title (i.e. **2.x** vs **3.x**)
- [ ] Check all commits' messages are clear, describing what and why vs how.
- [ ] Make sure **to have added unit tests or integration tests** to cover the new/modified code.
- [ ] Check if documentation is impacted by this change.

Please review the [guidelines for contributing](../CONTRIBUTING.md) and [Pull Request Instructions](https://github.com/aws/aws-parallelcluster/wiki/Git-Pull-Request-Instructions).

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: cli/tests/pcluster/templates/test_cw_dashboard_builder.py [hunk 1]
```diff
@@ -190,20 +190,25 @@ def _verify_common_error_metrics_graphs(cluster_config, output_yaml):
...
-            assert_that(output_yaml).does_not_contain("CannotRetrieveCustomScript")
-            assert_that(output_yaml).does_not_contain("ErrorWithCustomScript")
+            for metric in custom_script_metrics:
+                assert_that(output_yaml).does_not_contain(metric)
     else:
-        for metric in slurm_related_metrics:
+        assert_that(output_yaml).does_not_contain("Cluster Health Metrics")
+        for metric in slurm_related_metrics + custom_script_metrics:
+            assert_that(output_yaml).does_not_contain(metric)
             assert_that(output_yaml).does_not_contain(metric)
```
Comment (davprat) [inline cli/tests/pcluster/templates/test_cw_dashboard_builder.py]: Are these two lines the same?

**Review by davprat — DISMISSED** [review]
I'm probably mistaken, but it looks like a line is repeated twice in one of the tests.

File: cli/tests/pcluster/templates/test_cw_dashboard_builder.py (hunk 1 above)
Comment (chenwany) [reply cli/tests/pcluster/templates/test_cw_dashboard_builder.py]: Good catch! Removed one

**Review by davprat — DISMISSED** [review]
Lgtm.

**Review by davprat — DISMISSED** [review]

**Review by davprat — DISMISSED** [review]

**Review by davprat — APPROVED** [review]

**Pull request merged**